charts/console/values.yaml

```diff
@@ -775,3 +775,90 @@ dashboard:
       # ServiceMonitor connection tlsConfig. Defaults to {insecureSkipVerify:true}.
       tlsConfig:
         insecureSkipVerify: true
+
+
+cloudQuery:
+  replicaCount: 1
```

Comment:
we should add an enabled flag here, which would drop at minimum the deployment + service resources